src/core/src/sim/mc/moves/resize.rs

```diff
@@ -54,12 +54,23 @@ impl MCMove for Resize {
 
         // Get the largest cutoff of all intermolecular interactions in the
         // system.
-        // TODO: include electrostatic interactions
-        self.rc_max = system.interactions()
+
+        // Go through global interactions
+        let rc_glob = system.interactions()
+                            .globals()
+                            .iter()
+                            .map(|i| i.cutoff())
+                            .filter_map(|rc| rc)
+                            .fold(f64::NAN, f64::max);
```

Comment:
The check does the following:
```rust
if self.new_system.cell()                                                                                                                                       
            .lengths()
            .iter()
            .any(|&d| 0.5 * d <= self.rc_max) {
                fatal_error!("Tried to decrease the cell size but new size
                conflicts with the cut off radius. \
                Increase the number of particles to get rid of this problem.")
           }
```
If I understand the behavior of `NAN` correctly, every f64 (negative as well as positive) should be larger than `NAN`. So even if `self.rc_max = f64::NAN`, the error should not be triggered. But that is not very transparent. We could skip the whole if case by checking `if self.rc_max.is_nan`.

Am I getting this right? What do you think?